amavis/10-ask_peekaboo

```diff
@@ -99,31 +95,114 @@ sub ask_peekaboo_internal {
       $pmi->{$field} = $val if defined($val);
     }
 
-    push(@$metainfo, $pmi);
+    my $answer = $ua->post($scan_url,
+      Content_Type => 'form-data',
+      # could be per-file but complicates handling server-side
+      'X-Content-Disposition' => $pmi->{"content_disposition"},
+      Content => [
+        'file' => [
+          $pmi->{"full_name"},
```

Comment:
When testing with 'amavis' and a file called 'äußerst gemeiner Dateiname.doc'

```
Jan 17 17:40:16 peekabooav peekaboo[2300]: sanic.error - (MainThread) - ERROR - Failed when parsing form
Jan 17 17:40:16 peekabooav peekaboo[2300]: Traceback (most recent call last):
Jan 17 17:40:16 peekabooav peekaboo[2300]:   File "/opt/peekaboo/lib/python3.8/site-packages/sanic/request.py", line 367, in form
Jan 17 17:40:16 peekabooav peekaboo[2300]:     self.parsed_form, self.parsed_files = parse_multipart_form(
Jan 17 17:40:16 peekabooav peekaboo[2300]:   File "/opt/peekaboo/lib/python3.8/site-packages/sanic/request.py", line 780, in parse_multipart_form
Jan 17 17:40:16 peekabooav peekaboo[2300]:     form_line = form_part[line_index:line_end_index].decode("utf-8")
Jan 17 17:40:16 peekabooav peekaboo[2300]: UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe4 in position 55: invalid continuation byte
Jan 17 17:40:16 peekabooav peekaboo[2300]: peekaboo.server - (MainThread) - ERROR - File missing from request
```

I have good results replacing line 92 with this:
`$val = decode("UTF-8", $field) if $field eq "name_declared";`

Reply:
I will mark this resolved for this PR and keep it on my list

Reply:
Review reveals that our switch to libwww-perl and LWP::UserAgent actually broke encoding. I think we need to fix that before we can merge. I have some code that I hope I can push shortly.

Reply:
Sorry this took so long, but I've now pushed that cade that should improve the situation somewhat. The above problem and a number of others should be fixed. I think this is the best we can do for now.